### DISCUSSION TITLE
what does the status line mean?

### DISCUSSION THREAD

**Discussion by ghistes** [andmarti1424/sc-im#828] [Q&A]
Ok, I cannot figure out what the status line means and there does not seem to be any documentation...

When I open a file "hubba.sc" I get this status line at the very top:

hubba.sc; {Sheet1} A0 (10 2 0)

So it displays the filename, the sheet-name, the current cell and this mysterious "(10 2 0)" - what do these numbers mean?

**Comment by andmarti1424** [comment]
basically is the column format. column width, decimal precision and real format.
